yarn.lock

```diff
@@ -6892,6 +6883,13 @@ supports-color@^7.1.0:
   dependencies:
     has-flag "^4.0.0"
 
+swr@^0.3.7:
```

Comment:
remove all swr dependencies since we won't be using them to do any fetch requests